src/components/ui/InputText/input-text.scss

```diff
@@ -1,27 +1,71 @@
 @import "src/styles/scaffold";
 
 [data-fs-input-text] {
-  --fs-input-text-padding-horizontal: var(--fs-spacing-2);
+  // --------------------------------------------------------
+  // Design Tokens for Input Text
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-input-text-padding               : var(--fs-spacing-2) var(--fs-spacing-2) 0;
+  --fs-input-text-padding-label         : 0 var(--fs-spacing-2);
+
+  --fs-input-text-box-shadow            : none;
+  --fs-input-text-box-shadow-hover      : 0 0 0 var(--fs-border-width-default) var(--fs-border-color-default-active);
+
+  --fs-input-text-border-radius         : var(--fs-border-radius-default);
+  --fs-input-text-border-width          : var(--fs-border-width-default);
+  --fs-input-text-border-color          : var(--fs-border-color-default);
+  --fs-input-text-border-color-hover    : var(--fs-border-color-default-active);
+
+  --fs-input-text-text-color            : var(--fs-color-text);
+  --fs-input-text-text-color-label      : var(--fs-color-text-light);
+  --fs-input-text-text-size             : var(--fs-text-size-body);
+  --fs-input-text-text-size-message     : var(--fs-text-size-legend);
+  --fs-input-text-text-size-label       : var(--fs-text-size-tiny);
+  --fs-input-text-text-line-height      : 1.25;
+
+  --fs-input-text-transition-function   : var(--fs-transition-function);
+  --fs-input-text-transition-property   : var(--fs-transition-property);
+  --fs-input-text-transition-timing     : var(--fs-transition-timing);
+
+  // Error
+  --fs-input-text-error-border-color    : var(--fs-color-danger-border);
+  --fs-input-text-error-box-shadow      : 0 0 0 var(--fs-border-width-default) var(--fs-input-text-error-border-color);
+  --fs-input-text-error-focus-ring      : var(--fs-color-focus-ring-danger);
+
+  // Disabled
+  --fs-input-text-disabled-bkg-color    : var(--fs-color-disabled-bkg);
+  --fs-input-text-disabled-text-color   : var(--fs-color-disabled-text);
+  --fs-input-text-disabled-border-width : var(--fs-border-width-default);
+  --fs-input-text-disabled-border-color : var(--fs-border-color-default);
+
+  // --------------------------------------------------------
+  // Structural Styles
+  // --------------------------------------------------------
 
   position: relative;
   display: flex;
   flex-flow: column;
 
   input,
   label {
-    transition: var(--fs-transition-property) var(--fs-transition-timing) var(--fs-transition-function);
+    transition: var(--fs-input-text-transition-property) var(--fs-input-text-transition-timing) var(--fs-input-text-transition-function);
   }
 
   input {
     height: var(--fs-control-tap-size);
```

Comment:
wouldn't it be a local token too?

Reply:
Didn't get your question

Reply:
GH didn't highlight my point, sorry. It's about the `height` attribute, I was wondering if wouldn't be the case to create a local token for this